### PR TITLE
Adds a 'Add App Engine framework support' menu item - UI only

### PR DESCRIPTION
**Pull request by nbashirbello**
Partial fix for #1685

### CODE REVIEW

**Comment by etanshaul** [comment]
could you post some screenshots?

**Comment by nbashirbello** [comment]
addfacet1

**Comment by nbashirbello** [comment]
Based on what direction we decide to go, I'll rename the class to reflect facets or framework

**Comment by etanshaul** [comment]
How about keeping it higher level: "Enable App Engine support". Basically hide "framework" / "facet" from the user.

**Comment by patflynn** [comment]
Do we need the logos in the second submenu?

@etan are you referring to the menu item label? If so I'm a little unsure
about using 'enable' since that doesn't necessary feel like it would be
adding to or modifying the project.  For instance it could be read as
enabling app engine support in the plugin. (Unlikely I know, but users with
no context could be confused)

**Comment by etanshaul** [comment]
Fair enough. How about just “Add App Engine support” then?

Also I agree that the sub menus could be icon less.

**Comment by patflynn** [comment]
works for me.

**Comment by nbashirbello** [comment]
Updated image
screen shot 2017-10-13 at 2 03 25 pm

File: google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties [hunk 1]
```diff
@@ -202,6 +202,8 @@ appengine.run.settings.loglevel.label=Application log level:
 appengine.run.settings.adminhost.label=Admin host:
 appengine.run.settings.adminport.label=Admin port:
 appengine.infopanel.noproject=Can't retrieve region for selected project.
+appengine.add.standard.facet.tools.menu.description=Adds the Google App Engine Standard framework support to a module.
```
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: nit: remove the "the" after "Adds"

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -463,6 +463,10 @@
     </action>
 
     <group id="GoogleCloudTools" text="Google Cloud Tools" class="com.google.cloud.tools.intellij.CloudToolsMenuGroupAction" popup="true">
+      <group id="AddAppEngineFacet" text="Add App Engine support" class="com.google.cloud.tools.intellij.appengine.facet.AddAppEngineFacetMenuGroupAction" popup="true">
```
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: I guess this text can't be moved to the message bundle?

File: google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties (hunk 1 above)
Comment (etanshaul) [reply google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: Standard should not be capitalized. Same for Flexible in the next message.

**Comment by etanshaul** [comment]
lgtm, we should probably comment out the extension points in plugin.xml until the action is implemented

**Pull request merged**